### ISSUE TITLE
Video in README or website?

### ISSUE THREAD

**Issue by RossComputerGuy** [donovanglover/hyprnome#77]
Could we have a video or some sort of visual example of Hyprnome in action in the README or website (preferably both)?

**Comment by nickheyer** [comment]
Agreed. No visual example(s) being included in a repo like this makes zero sense. The developer's essentially asking people to just wing it, which makes me doubt the quality of the actual code that exists in the repo.
